docs/ios/bazel.md

```diff
@@ -0,0 +1,100 @@
+# iOS Build Instructions - Bazel
+
+## 1. Copy `productscience.yaml` to your project's root directory
+
+Product Science should provide you with a `productscience.yaml` file that contains your credentials and configuration details. If you never received this file or need to request a new copy, please contact support@productscience.ai.
```

Comment:
Not sure what contact info to provide here. I guessed at "support@productscience.ai" but don't know if that's a valid email address.

Reply:
Please ask @NateCham

Reply:
Please contact Nate

Reply:
support@productscience.ai works, I would also add Slack "please contact your designated integration support via previously agreed upon email/shared Slack channel or support@productscience.ai if neither of those are available"

Reply:
👍  updated.